src/components/dialog/dialog.js

```diff
@@ -1048,7 +1048,7 @@ function MdDialogProvider($$interimElementProvider) {
        */
       options.hideBackdrop = function hideBackdrop($destroy) {
         if (options.backdrop) {
-          if ( !!$destroy ) options.backdrop.remove();
+          if ( $destroy ) options.backdrop.remove();
```

Comment:
[ESLint](https://eslint.org/docs/rules/no-implicit-coercion) says that `!!` should be replaced by `Boolean($destroy)` for this rule. Is there a reason that you don't think that guidance applies here and in the other instances of this replacement in the PR?

Reply:
This was autofixed. It's because there's no need to coerce with `if ( ... )`

Reply:
also no-implicit-coercion is not enabled in eslint:recommended https://eslint.org/docs/rules/